test/AutomationNotes.MD

```diff
@@ -0,0 +1,38 @@
+# AUTOMATION NOTES:
```

Comment:
This seems like a weird place for this information. `CONTRIBUTING.md` seems like a weird choice too, but less weird than here. 

Also, capitalized file extensions are poor form.